<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_TimerApp_MainWindow" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget_TimerApp_MainWindow
{
    background-color:rgb(44, 44, 44)
}</string>
   </property>
   <widget class="QWidget" name="widget_TimerPopUp" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_TimerPopUp
{
   background-color: rgb(46, 46, 46);
   border-radius:  20px;
   z-index = 5;
}</string>
    </property>
    <widget class="QLabel" name="labelRest">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>431</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light SemiCondensed</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>REST for 1 minute</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_Time">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>491</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar#progressBar_Time
{
    border-radius:5px;
    text-align: center;
    
	font: 18pt &quot;MS Shell Dlg 2&quot;;
   
}

QProgressBar#progressBar_Time::chunk
{
   background-color:rgb(170, 170, 255)
}</string>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_skip">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>200</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_skip
{
   background-color: rgb(85, 170, 255);
   color:black;
	font: 25 24pt &quot;Bahnschrift Light SemiCondensed&quot;;
   border-radius: 20px;
}

QPushButton#pushButton_skip::pressed
{
   background-color: rgb(75, 150, 225);
}

QPushButton#pushButton_skip::hover
{
   border: 3px solid white;
}

QPushButton#pushButton_skip::disabled
{
   background-color: rgb(173, 173, 173);
   color:rgb(199, 199, 199);
}</string>
     </property>
     <property name="text">
      <string>Skip</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1MinSnooze">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>230</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_1MinSnooze
{
   background-color: rgb(85, 170, 255);
   color:black;
	font: 25 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
   border-radius: 10px;
}

QPushButton#pushButton_1MinSnooze::pressed
{
   background-color: rgb(75, 150, 225);
}

QPushButton#pushButton_1MinSnooze::hover
{
   border: 3px solid white;
}

QPushButton#pushButton_1MinSnooze::disabled
{
   background-color: rgb(173, 173, 173);
   color:rgb(199, 199, 199);
}
</string>
     </property>
     <property name="text">
      <string>Snooze 
1 min</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5MinSnooze">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>220</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_5MinSnooze
{
   background-color: rgb(85, 170, 255);
   color:black;
	font: 25 12pt &quot;Bahnschrift Light SemiCondensed&quot;;
   border-radius: 10px;
}

QPushButton#pushButton_5MinSnooze::pressed
{
   background-color: rgb(75, 150, 225);
}

QPushButton#pushButton_5MinSnooze::hover
{
   border: 3px solid white;
}

QPushButton#pushButton_5MinSnooze::disabled
{
   background-color: rgb(173, 173, 173);
   color:rgb(199, 199, 199);
}
</string>
     </property>
     <property name="text">
      <string>Snooze 
5 min</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_TimerPopup_Close">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>0</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Light SemiCondensed</family>
       <pointsize>16</pointsize>
       <weight>3</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pushButton_TimerPopup_Close
{
   background-color: rgb(0, 0, 0);
   border-radius: 5px;
   color: white;
   font: 25 16pt &quot;Bahnschrift Light SemiCondensed&quot;;
}

QPushButton#pushButton_TimerPopup_Close::pressed
{
   background-color: rgb(216, 81, 86);
   
}

QPushButton#pushButton_TimerPopup_Close::hover
{
   background-color: rgb(253, 95, 103);
   
}</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1041</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_5</zorder>
    <zorder>labelRest</zorder>
    <zorder>progressBar_Time</zorder>
    <zorder>pushButton_skip</zorder>
    <zorder>pushButton_1MinSnooze</zorder>
    <zorder>pushButton_5MinSnooze</zorder>
    <zorder>pushButton_TimerPopup_Close</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
